prerendering.bs

```diff
@@ -658,6 +673,37 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
```

Comment:
Nit: "prerender-scoped" is a bit more grammatical than "prerender scoped"

Reply:
renamed it and referrences below, Thanks!